ui/main.js

```diff
@@ -0,0 +1,32 @@
+const pug = require('pug');
+const fs = require('fs');
+const path = require('path');
+const yargs = require('yargs');
+const chokidar = require('chokidar');
+
+const { argv } = yargs
+  .option('watch', {
+    alias: 'w',
+    description: 'Watch for pug changes',
+  })
+  .help();
+
+function renderHtml() {
+  const html = pug.renderFile(`${__dirname}/main.pug`, { pretty: '  ' });
```

Comment:
`pug` has deprecated `pretty` option, but this helps keep the output HTML legible.
We could also look into post-processing to prettify the HTML.